doc/sphinx/credits.rst

```diff
@@ -1404,3 +1404,141 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
+deprecations that are documented in ``CHANGES.md`` and new features that
+are documented in the reference manual. Here are the most important
+changes:
+
+- Kernel: mutually defined records are now supported, by Pierre-Marie
+  Pédrot.
+
+- Notations:
+
+  - Support for autonomous grammars of terms called "custom entries", by
+    Hugo Herbelin (see section :ref:`custom-entries` of the reference
+    manual).
+
+  - Deprecated notations of the standard library will be removed in the
+    next version of |Coq|, see the ``CHANGES.md`` file for a script to
+    ease porting, by Jason Gross.
+
+  - Added the :cmd:`Numeral Notation` command for registering decimal
+    numeral notations for custom types, by Pierre Letouzey and Jason
+    Gross.
+
+- Tactics: Introduction tactics :tacn:`intro`/:tacn:`intros` on a goal that is an
+  existential variable now force a refinement of the goal into a
+  dependent product rather than failing, by Hugo Herbelin.
+
+- Decision procedures: deprecation of tactic ``romega`` in favor of
+  :tacn:`lia` and removal of ``fourier``, replaced by :tacn:`lra`, by
+  Maxime Dénès.
+
+- Proof language: focusing bracket ``{`` now supports named
+  :ref:`goals <curly-braces>`, e.g. ``[x]:{`` will focus
+  on a goal (existential variable) named ``x``, by Théo Zimmermann.
+
+- SSReflect: delayed clear switches have a simpler semantics and empty
+  clear switches are allowed before view applications of variable views
+  (see section :ref:`introduction_ssr`), by Enrico Tassi.
+
+- Vernacular:
+
+  - Experimental support for :ref:`attributes <gallina-attributes>` on
+    commands, by Vincent Laporte, as in ``#[local] Lemma foo : bar.``.
+    Tactics and tactic notations now support the ``deprecated``
+    attribute.
+
+  - Removed deprecated commands ``Arguments Scope`` and ``Implicit
+    Arguments`` in favor of :cmd:`Arguments`.
+
+  - New option :flag:`Uniform Inductive Parameters` by Jesper Hugunin to
+    avoid repeating uniform parameters in constructor declarations.
+
+  - New commands :cmd:`Hint Variables` and :cmd:`Hint Constants`, by
+    Matthieu Sozeau, for controlling the opacity flag of variables and
+    constants in hint databases.
+
+  - Multiple sections with the same name are now allowed.
+
+- Library: additions and changes in the ``VectorDef``, ``Ascii`` and
+  ``String`` library. Syntax notations are now available only when using
+  ``Import`` of libraries and not merely ``Require``.
+
+- Toplevels: ``coqtop`` and ``coqide`` can now display diffs between proof
+  steps in color, using the ``Set Diffs`` option, by Jim Fehrle (see
+  section :ref:`showing_diffs`).
+
+- Documentation: we integrated a large number of fixes to the new Sphinx
+  documentation.
+
+- Tools: removed the ``gallina`` utility and the homebrewed ``Emacs`` mode.
+
+- Packaging: Michael Soegtrop maintains a compilation chain for Windows
+  allowing to build Coq with optional packages, including Mtac2, Ltac2 and
+  Equations for this release.
```

Comment:
... and many other packages.

Reply:
What do you mean? The sentence means with "any" optional package but those ones are part of the release.

Reply:
There are more than these three in the release (QuickChick, math-comp, etc.). BTW, this work was also included in 8.8.2.

Reply:
Ok, I didn't know. I fixed this accordingly, just reminding:
```
- Packaging: as in |Coq| 8.8.2, the Windows installer now includes many
  more external packages that can be individually selected for
  installation, by Michael Soegtrop.
```